### PR TITLE
UPDATING 0V90.JSON

### PR DESCRIPTION
**Pull request by ghost**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/0v90.json [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
+        
     },
     "record": {
-        "A": [
-            "35.200.171.190",
-            "35.240.84.96",
-            "34.132.146.171"
-        ]
+        "CNAME": "0v90.github.io",
+        "TXT": "_github-pages-challenge-0v90",
```
Comment (CuteDog5695) [inline domains/0v90.json:12]: You cannot use CNAME with TXT (Follow the docs here on how to do the txt verification https://www.is-a.dev/docs/github-pages-verification/)

File: domains/0v90.json [hunk 2]
```diff
@@ -1,14 +1,16 @@
 {
+    "description": "Personal Portfolio Website",
+    "repo": "https://github.com/0v90/0v90.github.io",
     "owner": {
         "username": "0v90",
+        "discord": 651519394673065989,
         "email": "",
```
Comment (CuteDog5695) [inline domains/0v90.json:7]:
```suggestion
        "discord": "651519394673065989",
        "email": ""
```

File: domains/0v90
Comment (CuteDog5695) [inline domains/0v90:1]: File needs to be json, but since its technically a duplicate this should be deleted.

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/_github-pages-challenge-0v90.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "0v90",
+        "email": ""
```
Comment (CuteDog5695) [inline domains/_github-pages-challenge-0v90.json:4]: Please add any forms of contact (Like Discord, Twitter, Mastodon, Reddit etc.)

File: domains/0v90.json [hunk 4]
```diff
@@ -1,14 +1,16 @@
 {
+    "description": "Personal Portfolio Website",
+    "repo": "https://github.com/0v90/0v90.github.io",
     "owner": {
         "username": "0v90",
-        "email": "",
-        "Discord": "0v90"
+        "discord": 651519394673065989,
```
Comment (CuteDog5695) [inline domains/0v90.json:6]:
```suggestion
        "discord": "651519394673065989",
```

**Review by iostpa — CHANGES_REQUESTED** [review]

**Review by iostpa — APPROVED** [review]
Seems fine to me now

**Pull request merged**